installation/scripts/minikube.sh

```diff
@@ -170,19 +180,36 @@ function start() {
         fixDindMinikubeIssue
     fi
 
-    minikube start \
-    --memory 8192 \
-    --cpus 4 \
-    --extra-config=apiserver.Authorization.Mode=RBAC \
-    --extra-config=apiserver.GenericServerRunOptions.CorsAllowedOriginList=".*" \
-    --extra-config=controller-manager.ClusterSigningCertFile="/var/lib/localkube/certs/ca.crt" \
-	--extra-config=controller-manager.ClusterSigningKeyFile="/var/lib/localkube/certs/ca.key" \
-    --extra-config=apiserver.Admission.PluginNames="LimitRanger,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota" \
-    --kubernetes-version=v$KUBERNETES_VERSION \
-    --vm-driver=$VM_DRIVER \
-    --feature-gates="MountPropagation=false" \
-    -b=localkube
-
+    if [[ $KUBEADM ]]; then
```

Comment:
Lot's of repetition here. It's hard to understand the differences. Please extract all common properties to some variable, and put only the differences in the conditional.